lib/ansible/modules/cloud/openstack/os_group_info.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+
+# Copyright (c) 2019, Phillipe Smith <phillipelnx@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: os_group_info
+short_description: Retrieve info about one or more OpenStack groups
+extends_documentation_fragment: openstack
+version_added: "2.9"
+author: "Phillipe Smith (@phsmith)"
+description:
+    - Retrieve info about a one or more OpenStack groups
+requirements:
+    - "python >= 2.7"
+    - "openstacksdk"
+options:
+   name:
+     description:
+        - Name or ID of the group
+     required: true
+     type: str
+   domain:
+     description:
+        - Name or ID of the domain containing the group if the cloud supports domains
+     type: str
+   filters:
+     description:
+        - A dictionary of meta data to use for further filtering.  Elements of
+          this dictionary may be additional dictionaries.
+     type: dict
+   availability_zone:
+     description:
+       - Ignored. Present for backwards compatibility
```

Comment:
Does it make sense to have this in a new module?

Reply:
Don't thing so. Removed that option.